README.md

```diff
@@ -344,25 +344,25 @@ Please take a quick gander at the [contribution guidelines](https://github.com/a
     * [asc](https://github.com/viant/asc) - Datastore Connectivity for Aerospike for go.
     * [cayley](https://github.com/google/cayley) - Graph database with support for multiple backends.
     * [dsc](https://github.com/viant/dsc) - Datastore connectivity for SQL, NoSQL, structured files.
-    * [dynago](https://github.com/underarmour/dynago) - Dynago is a principle of least surprise client for DynamoDB
-    * [go-couchbase](https://github.com/couchbase/go-couchbase) - Couchbase client in Go
-    * [go-couchdb](https://github.com/fjl/go-couchdb) - Yet another CouchDB HTTP API wrapper for Go
-    * [gocb](https://github.com/couchbase/gocb) - Official Couchbase Go SDK
+    * [dynago](https://github.com/underarmour/dynago) - Dynago is a principle of least surprise client for DynamoDB.
+    * [go-couchbase](https://github.com/couchbase/go-couchbase) - Couchbase client in Go.
+    * [go-couchdb](https://github.com/fjl/go-couchdb) - Yet another CouchDB HTTP API wrapper for Go.
+    * [gocb](https://github.com/couchbase/gocb) - Official Couchbase Go SDK.
     * [gocql](http://gocql.github.io) - Go language driver for Apache Cassandra.
     * [gomemcache](https://github.com/bradfitz/gomemcache/) - memcache client library for the Go programming language.
-    * [gorethink](https://github.com/dancannon/gorethink) - Go language driver for RethinkDB
-    * [goriak](https://github.com/zegl/goriak) - Go language driver for Riak KV
+    * [gorethink](https://github.com/dancannon/gorethink) - Go language driver for RethinkDB.
+    * [goriak](https://github.com/zegl/goriak) - Go language driver for Riak KV.
     * [mgo](https://godoc.org/labix.org/v2/mgo) - MongoDB driver for the Go language that implements a rich and well tested selection of features under a very simple API following standard Go idioms.
-    * [neo4j](https://github.com/cihangir/neo4j) - Neo4j Rest API Bindings for Golang
+    * [neo4j](https://github.com/cihangir/neo4j) - Neo4j Rest API Bindings for Golang.
     * [Neo4j-GO](https://github.com/davemeehan/Neo4j-GO) - Neo4j REST Client in golang.
-    * [neoism](https://github.com/jmcvetta/neoism) - Neo4j client for Golang
+    * [neoism](https://github.com/jmcvetta/neoism) - Neo4j client for Golang.
     * [redigo](https://github.com/garyburd/redigo) - Redigo is a Go client for the Redis database.
-    * [redis](https://github.com/go-redis/redis) - Redis client for Golang
+    * [redis](https://github.com/go-redis/redis) - Redis client for Golang.
     * [redis](https://github.com/hoisie/redis) - Simple, powerful Redis client for Go.
     * [redis](https://github.com/bsm/redeo) - Redis-protocol compatible TCP servers/services.
     * [xredis](https://github.com/shomali11/xredis) - Typesafe, customizable, clean & easy to use Redis client.
 
-* Search and Analytic Databases
+* Search and Analytic Databases.
```

Comment:
This is more of a title rather than a sentence, so it doesn't need a period. "Relational Databases" above is another title that doesn't have a period.